spec/tapioca/cli_spec.rb

```diff
@@ -451,47 +452,50 @@ def title=(title); end
       assert_path_exists("#{outdir}/namespace/comment.rbi")
       refute_path_exists("#{outdir}/user.rbi")
 
-      assert_equal(<<~CONTENTS.chomp, File.read("#{outdir}/baz/role.rbi"))
+      assert_equal(<<~CONTENTS, File.read("#{outdir}/baz/role.rbi"))
         # DO NOT EDIT MANUALLY
         # This is an autogenerated file for dynamic methods in `Baz::Role`.
         # Please instead update this file by running `bin/tapioca dsl Baz::Role`.
 
         # typed: true
-        module Baz
-          class Role
-            sig { returns(T.nilable(::String)) }
-            def title; end
 
-            sig { params(title: T.nilable(::String)).returns(T.nilable(::String)) }
-            def title=(title); end
-          end
+        class Baz::Role
+          sig { returns(T.nilable(::String)) }
+          def title; end
+
+          sig { params(title: T.nilable(::String)).returns(T.nilable(::String)) }
+          def title=(title); end
         end
       CONTENTS
 
-      assert_equal(<<~CONTENTS.chomp, File.read("#{outdir}/job.rbi"))
+      assert_equal(<<~CONTENTS, File.read("#{outdir}/job.rbi"))
         # DO NOT EDIT MANUALLY
         # This is an autogenerated file for dynamic methods in `Job`.
         # Please instead update this file by running `bin/tapioca dsl Job`.
 
         # typed: true
+
         class Job
-          sig { params(foo: T.untyped, bar: T.untyped).returns(String) }
-          def self.perform_async(foo, bar); end
+          class << self
```

Comment:
Why aren't these methods generated as `self.foo` anymore? I thought DSL generation was not doing `class << self` for class methods.

Reply:
My bad, I added the `nest_singleton_methods` rewriter to the DSL output:  https://github.com/Shopify/tapioca/pull/388/files#diff-5f22751f990a0340da79d3ef08e0c2785b41d8516748fc7fc3c20851b5bc3c28R91 I'll remove it.

It is not used in the `dsl_spec` tho, hence why those are not rewritten in the DSL specs like Kaan noticed: https://github.com/Shopify/tapioca/pull/388/files#diff-64b06a679c19070519b7f3394c9ba6d1a8c5c8d44c4bf61a9f13dc832f9b9a46R80-R83

Reply:
Yeah, that's how I noticed the difference in behaviour. :)